### DISCUSSION TITLE
Count Number Position

### DISCUSSION THREAD

**Discussion by RandalMAymond** [roboflow/supervision#326] [Q&A]
How am I able to change the position of the count number? Its current in the center of my polygon however I'd like to change that position.

**Comment by SkalskiP** [comment]
I'm afraid you can't do it now unless you fork `supervision` and overwrite the current behavior. Where would you like to plot the count?

**Reply by RandalMAymond** [reply]
Preferably a corner. With it being in the center it covers some of the objects I'm trying to count.

**Reply by SkalskiP** [reply]
Is it the corner of the frame or the corner of the zone?

**Reply by RandalMAymond** [reply]
Either would be an acceptable solution.
